pages/_/components.tsx

```diff
@@ -55,28 +55,28 @@ export default function DesignPage() {
       <Section>
         <TitleM>ButtonLink</TitleM>
         <Examples>
-          <ButtonLink href="#">primary</ButtonLink>
-          <ButtonLink href="#" pending>
+          <ButtonLink href={{ pathname: '/' }}>primary</ButtonLink>
+          <ButtonLink href={{ pathname: '/' }} pending>
```

Comment:
@tbezman Would that work? or do we need a workaround here?

Reply:
No problem here